app/views/moderations/index.html.erb

```diff
@@ -31,12 +31,15 @@
   </main>
 <% else %>
   <div class="container" style="margin-top: 90px;">
-    <h1 style="text-align: center"><%= community_name %> Mods</h1>
+    <h1 class="pt-5" style="text-align: center"><%= community_name %> Mods</h1>
     <div class="body">
       <p>We periodically award some <%= community_name %> members with heightened privileges to help moderate the community.</p>
-      <p>Email <%= email_link %> if you'd like to be considered right away.</p>
+      <% if SiteConfig.dev_to? %>
+        <p>Check out our <a href="https://dev.to/code-of-conduct">Code of Conduct</a> and read through our <a href="https://dev.to/community-moderation">Trusted User Guide</a> and <a href="https://dev.to/tag-moderation">Tag Moderation Guide</a>.</p>
```

Comment:
Is there any way we can generalize this code so we don't have to use `if SiteConfig.dev_to?`?

Reply:
Great question @mstruve . I thought about this for some time. The thing is, those links are specific to DEV. I checked if we have any SiteConfig variables that contain custom links like these, but we have none as yet. I think it's a great idea though to add to our Forem generalization goals 😄 

I'm open to any suggestions from you though!

Reply:
@msarit I think you should be able to link to `/community-moderation` and `/tag-moderation`. It looks like those routes exist on forem.team and forem.dev, and they exist as routes in the `routes.rb` file right now:

https://github.com/forem/forem/blob/27648a938b0168b2b9fc2cd8162b203e44d8100d/config/routes.rb#L418-L421

So, you should be able to link without hardcoding to dev.to (and then you don't need to use `dev_to?` 😄). Ideally, we want to remove the `dev_to?` helper, so we should avoid using that since we know we want to move towards generalizing our codebase entirely :)

Reply:
@vaidehijoshi ok this sounds good. my only question is: are all forems going to have these three links at all times? `/community-moderation`, `/tag-moderation` and `/code-of-conduct`?

Reply:
I suspect we will always have a CoC, but I am not 100% sure about the others. I will say that we have these links throughout our site right now; [see all these CoC occurrences](https://github.com/forem/forem/search?q=code-of-conduct), [tag moderation occurrences](https://github.com/forem/forem/search?q=tag-moderation), and [community-moderation occurrences](https://github.com/forem/forem/search?q=community-moderation). So linking to them directly is following a pattern we already have set in the codebase. If we decide to move away from these links, we're going to have to address all of these occurrences, so I would say that if we _do_ decide not to have these links on every Forem, we'll have to address is holistically.

TL;DR, we have enough links to these 3 pages in the codebase, so I think it's okay to add another one in an effort to not hardcode DEV-specific things :)

Reply:
@vaidehijoshi Np! done!